packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx

```diff
@@ -89,36 +108,60 @@ const AlbumCard = ({ album, index, isLoading, setDidLoad }: AlbumCardProps) => {
       cardCoverImageSizes={album._cover_art_sizes}
       onClick={handleClick}
     />
-  )
+  ) : null
 }
 
 const AlbumsTabContent = () => {
   const goToRoute = useGoToRoute()
 
-  // Temporarily requesting large page size to ensure we get all albums
-  // until the list is updated to use `InfinteScroll`
-  const { data: albums } = useSavedAlbumsDetails({ pageSize: 9999 })
-  const { isLoading, setDidLoad } = useOrderedLoad(albums.length)
-  const cards = albums.map((album, i) => {
+  const { data: savedAlbums } = useSavedAlbums()
```

Comment:
I wonder if `useSavedAlbums` could accept a selector so we could avoid the extra `useMemo`?

```suggestion
  const savedAlbumIds = useSavedAlbums((savedAlbums) => savedAlbums.map((a) => a.id))
```

Reply:
We'd have to memo it on the mobile component that allows you to filter the name (why don't we let you do this on desktop??) anyways, so IMO fine as is

Reply:
My follow-up branch is changing some of this due to the filtering we have to do on mobile. Agreed that  it's weird that we don't allow filtering on desktop. I actually think it would be pretty easy to add and may do that if it doesn't take too long.